### PR TITLE
Fix the dependencies missing in bibtex layer.

### PR DESCRIPTION
**Pull request by myme5261314**
#6091

### CODE REVIEW

**Comment by myme5261314** [comment]
#6014

File: layers/+lang/bibtex/packages.el [hunk 1]
```diff
@@ -71,8 +74,19 @@
         "li" 'isbn-to-bibtex
         "lp" 'pubmed-insert-bibtex-from-pmid))))
 
+(defun bibtex/post-init-org-ref ()
+  (add-hook 'org-mode-hook (lambda () (require 'org-ref)))
+  )
```
Comment (StreakyCobra) [inline layers/+lang/bibtex/packages.el]: Style: we keep closing parenthesis with the last line of code.

File: layers/+lang/bibtex/packages.el [hunk 2]
```diff
@@ -71,8 +74,19 @@
...
+
 (defun bibtex/post-init-markdown-mode ()
   (spacemacs/set-leader-keys-for-major-mode 'markdown-mode
     "ic" 'org-ref-helm-insert-cite-link))
 
+(defun bibtex/init-helm-bibtex ()
+  )
```
Comment (StreakyCobra) [inline layers/+lang/bibtex/packages.el]: Same here

File: layers/+lang/bibtex/packages.el [hunk 3]
```diff
@@ -71,8 +74,19 @@
...
+(defun bibtex/init-biblio ()
+  )
```
Comment (StreakyCobra) [inline layers/+lang/bibtex/packages.el]: Same here

File: layers/+lang/bibtex/packages.el [hunk 4]
```diff
@@ -71,8 +74,19 @@
...
+(defun bibtex/init-biblio-core ()
+  )
```
Comment (StreakyCobra) [inline layers/+lang/bibtex/packages.el]: Same here

**Comment by StreakyCobra** [comment]
I haven't checked in details, but you may also want to `(use-package …` the packages.

**Comment by myme5261314** [comment]
Will this be merged?

**Comment by syl20bnr** [comment]
Yes it will :-)
When the issue is resolved upstream we will be able to remove this work-around.
Thank you ! 👍 
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**